README.md

```diff
@@ -345,6 +345,21 @@ Fired when a fatal error occurs. Usually, this means bad signaling data was rece
 
 `err` is an `Error` object.
 
+## error codes
+
+To make debugging and logging errors easier, `Error` objects have an attached error code. The `err.code` property will indicate the origin of the failure and `err.message` will share further details on what went wrong.
+
+- `ERR_WEBRTC_SUPPORT` - The failure occurred in the webRTC support check
```

Comment:
The codes are pretty self-explanatory. I don't think this list is necessary.

Reply:
It is sometimes useful albeit somewhat tedious to maintain, to have a list of codes in readme rather than searching through source code.

Reply:
A simple list of codes might be useful, but these descriptions don't really add any explanation.

Reply:
I'm don't have strong feelings on this.  However the readme section was requested by @feross in the feature request thread (https://github.com/feross/simple-peer/issues/238).

Is there a way I can add more info there to make it more useful?

Happy to remove the section if that is the consensus.

Reply:
I've committed suggested changes to the README.